### PR TITLE
Fixed dependencies

### PR DESCRIPTION
**Pull request by Ap0ph1s-W4ite**
Fixed the Dockerfile with new dependencies and also fixed the package warning and URL 404 errors.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,20 +12,24 @@ RUN apt install bluez -y
 
 RUN apt install build-essential libglib2.0-dev libical-dev libreadline-dev libudev-dev libdbus-1-dev libdbus-glib-1-dev bluetooth libbluetooth-dev usbutils -y
 
-RUN pip3 install idasen-controller==2.0.1
+RUN export SKIP_CYTHON=false
```
Comment (maxime1992) [inline Dockerfile:16]: Can you please add a comment explaining why this is needed?

File: Dockerfile [hunk 2]
```diff
@@ -12,20 +12,24 @@ RUN apt install bluez -y
...
 
-RUN apt install curl -y
+RUN pip3 install idasen-controller==2.2.0
+
+RUN apt-get update
```
Comment (maxime1992) [inline Dockerfile:20]: Why the change to apt-get?

**Review by maxime1992 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Ap0ph1s-W4ite) [reply Dockerfile:16]: Was needed because of this [issue with dbus-fast](https://github.com/Bluetooth-Devices/dbus-fast/issues/237#issuecomment-1705993762).

It makes the wheel-build a lot faster.

File: Dockerfile (hunk 2 above)
Comment (Ap0ph1s-W4ite) [reply Dockerfile:19]: Old habits, sorry! apt is fine but needs the `apt update` command to fix the error 404 when apt looks for packages list.

Comment (Ap0ph1s-W4ite) [reply Dockerfile:20]: Have to check again, but I believe that's old habits.

**Review by maxime1992 — APPROVED** [review]

**Comment by maxime1992** [comment]
Thanks for the PR @Ap0ph1s-W4ite :+1: !
